data/variants.json

```diff
@@ -27281,5 +27281,17 @@
       "Prism"
     ],
     "showSuitNames": true
+  },
+  {
+    "name": "Sudoku",
+    "id": 1792,
+    "suits": [
+      "Red",
+      "Yellow",
+      "Green",
+      "Blue",
+      "Purple"
+    ],
+    "showSuitNames": true
```

Comment:
You're not supposed to edit this file directly since it's generated by this python script: https://github.com/Zamiell/hanabi-live/blob/master/scripts/python/create_variants_json.py